CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+
+Enhancement suggestions are tracked as [GitHub issues](https://github.com/kelektiv/node-cron/issues).
+
+- Use a **clear and descriptive title** for the issue to identify the suggestion.
+- Provide a **step-by-step description of the suggested enhancement** in as many details as possible.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why. At this point you can also tell which alternatives do not work for you.
+- **Explain why this enhancement would be useful** to most cron users. You may also want to point out the other projects that solved it better and which could serve as inspiration.
+
+## Working with the code
+
+### Set up the workspace
+
+[Fork](https://guides.github.com/activities/forking/#fork) the project, [clone](https://guides.github.com/activities/forking/#clone) your fork, configure the remotes and install the dependencies:
+
+```bash
+# Clone your fork of the repo into the current directory
+$ git clone git@github.com:<your-username>/node-cron.git # or https://github.com/<your-username>/node-cron.git for HTTPS
+# Navigate to the newly cloned directory
+$ cd node-cron
+# Assign the original repo to a remote called "upstream"
+$ git remote add upstream git@github.com:kelektiv/node-cron.git # or https://github.com/kelektiv/node-cron.git for HTTPS
+# Install the dependencies
+$ npm install
+```
+
+### Lint
+
+This repository uses [ESLint](https://github.com/eslint/eslint) and [Prettier](https://prettier.io) for linting and formatting.
+
+Before pushing your code changes make sure there are no linting errors with `npm run lint`.
+
+**Tips**:
+
+- Most linting errors can be automatically fixed with `npm run lint:fix`.
+- Install the [ESLint plugin](https://eslint.org/docs/latest/use/integrations) for your editor to see linting errors directly in your editor and automatically fix them on save.
+
+### Tests
+
+This repository uses [Jest](https://github.com/jestjs/jest) for writing and running tests.
+
+Before pushing your code changes make sure all **tests pass** and the **coverage thresholds are met**:
+
+```bash
+$ npm run test
+```
+
+**Tips:** During development you can:
+
+- run a single test file with `npm run test -- <glob>`, for example `npm run test -- tests/crontime.test.ts`
+- run a single test case with `npm run test -- -t 'test case name regex'`, for example `npm run test -- -t 'should accept all valid ranges'`
+- run in watch mode with `npm run test:watch` to automatically run a test case when you modify it or the associated source code
+
+## Join The Project Team
+
+This project is looking for help! If you're interested in helping with the project please reach out to me (ncb000gt) on [Twitter](https://twitter.com/ncb000gt). We'd love for it to continue on, but it needs a lot of attention. You can also join the [Discord server](https://discord.gg/yyKns29zch) to learn more about what needs to be done.
```

Comment:
at this point we might just want to remove the part about reaching out on Twitter. Discord is a better place to get new people oriented to contributing since any existing contributors can respond rather than a single person